docs/release-notes/template.md

```diff
@@ -0,0 +1,54 @@
+<!--
```

Comment:
please review this in https://github.com/improbable-eng/etcd-cluster-operator/pull/70

github won't let me base my PR on another branch in my own fork